### PR TITLE
Feature/pbs 1302 flight time card

### PR DESCRIPTION
**Pull request by tsanders88**

### CODE REVIEW

File: PBLogging/Extensions/CLLocationCoordinate2D+Ext.swift [hunk 1]
```diff
@@ -77,10 +77,10 @@ extension CLLocationCoordinate2D {
         let lon2 = location.longitude.toRadians
         
         let haversin = { (angle: Double) -> Double in
-            return (1 - cos(angle)) / 2
+            (1 - cos(angle)) / 2
         }
         let ahaversin = { (angle: Double) -> Double in
-            return 2 * asin(sqrt(angle))
+            2 * asin(sqrt(angle))
```
Comment (tsanders88) [inline PBLogging/Extensions/CLLocationCoordinate2D+Ext.swift:83]: unrelated, xcode keeps automatically changing when i load the project

Comment (jbethanco) [reply PBLogging/Extensions/CLLocationCoordinate2D+Ext.swift:83]: May be swiftlint ?

File: PBLogging/Components/CardContent.swift [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        }
+    }
+}
+
+struct ContentRow: View {
+    
+    let imageName: String
+    let title: String
+    let leftColumnLabels: [String]
+    let rightColumnLabels: [String]
```
[the remark this answers is not in the document]
Comment (jbethanco) [reply PBLogging/Components/CardContent.swift:43]: Also repeats the problem from ContentColumn with coordinated arrays needing to be passed in.

File: PBLogging/Components/CardContent.swift [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    var isBold = true
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            ForEach(labels, id: \.self) { label in
+                if isBold {
+                    Text(label).font(.pblBold(size: 14))
+                } else {
+                    Text(label).font(.pblRegular(size: 14))
+                }
```
Comment (jbethanco) [inline PBLogging/Components/CardContent.swift:84]: Don't think you wrote this... but modifiers should be on their own line.

**Review by jbethanco — COMMENTED** [review]
I think there was some weirdness from before your time that may have snuck in here.

File: PBLogging/Views/SingleSortie/MissionDataView.swift [hunk 4]
```diff
@@ -92,6 +91,46 @@ struct MissionDataView: View {
...
+                .frame(maxHeight: .infinity)
+                .background(Color.pblBackgroundDefault)
+                ContentRow(imageName: "person.circle", title: "Crew Positions", leftColumnLabels: crewPositions.map { $0.0 }, rightColumnLabels: crewPositions.map { $0.1 })
+                    .foregroundColor(.pblForegroundSecondary)
+                    .padding(.leading)
+                Spacer()
+            }
+            .frame(maxWidth: .infinity)
+        }
+    }
```
[the remark this answers is not in the document]
Comment (tsanders88) [reply PBLogging/Views/SingleSortie/MissionDataView.swift:123]: mainly reusing what was already there. contentrow is used in 1303 too

**Review by mcgarrys — APPROVED** [review]
Looks good and works 💥 👌

**Pull request merged**